editor.planx.uk/src/@planx/components/Pay/model.ts

```diff
@@ -10,12 +10,14 @@ import { array, boolean, object, string } from "yup";
 
 import { type BaseNodeData, parseBaseNodeData } from "../shared";
 
+export const PASSPORT_FN = "application.fee.payable" as const;
```

Comment:
nit: can we rename to `PAY_FN` so more clear if/when exported/imported beyond this component? (Eg tests, etc). Already have a few `DEFAULT_FN` etc that I notice that we're commonly re-naming when importing elsewhere.